.github/workflows/windows-packages.yaml

```diff
@@ -109,10 +119,13 @@ jobs:
 
     - name: Test creating extension
       run: |
-        ~/PostgreSQL/${{ matrix.pg }}/bin/psql -U postgres -d postgres -X -c "CREATE EXTENSION timescaledb;SELECT extname,extversion,version() FROM pg_extension WHERE extname='timescaledb';"
-        $installed_version = ~/PostgreSQL/${{ matrix.pg }}/bin/psql -U postgres -d postgres -X -t -c "SELECT extversion FROM pg_extension WHERE extname='timescaledb';"
+        ~/PostgreSQL/${{ matrix.pg }}/bin/psql -U postgres -d postgres -X `
+          -c "CREATE EXTENSION timescaledb" `
+          -c "SELECT extname,extversion,version() FROM pg_extension WHERE extname='timescaledb'"
+        $installed_version = ~/PostgreSQL/${{ matrix.pg }}/bin/psql -U postgres `
+          -d postgres -X -t -c "SELECT extversion FROM pg_extension WHERE extname='timescaledb';"
         $installed_version = $installed_version.Trim()
-        if (${installed_version} -notmatch "${{ steps.version.outputs.version }}")
+        if ("${installed_version}" -ne "${{ steps.version.outputs.version }}")
```

Comment:
Also fixing this check while I'm at it. Somehow the version was empty, and this check always returned true because it was using `-nomatch` which means "doesn't match a regular expression", which in turn is always false for an empty regex.